infrastructure/logging/src/main/java/tech/pegasys/teku/infrastructure/logging/ValidatorLogger.java

```diff
@@ -184,6 +184,15 @@ public void executionPayloadPreparedUsingBeaconDefaultFeeRecipient(final UInt64
             Color.YELLOW));
   }
 
+  public void executionPayloadPreparedUsingBurnAddressForFeeRecipient(final UInt64 slot) {
+    log.error(
+        ColorConsolePrinter.print(
+            "Producing block at slot "
+                + slot
+                + ", no proposer was prepared, and no default fee recipient defined, using 0x00 BURN address",
```

Comment:
```suggestion
            "Producing block at slot "
                + slot
                + ", no proposer was prepared, and no default fee recipient defined, inclusion fees will be lost",
```

not sure how well people understand the idea of a burn address so probably slightly better to just be clear about what will be lost.